internal/cli/atlas/clusters/create.go

```diff
@@ -71,15 +73,16 @@ func (opts *CreateOpts) initStore(ctx context.Context) func() error {
 	}
 }
 
-var createTmpl = "Deploying cluster '{{.Name}}'.\n"
+var createWatchTmpl = "Cluster '{{.Name}}' created successfully.\n"
+var clusterObj *atlasv2.ClusterDescriptionV15
```

Comment:
does this need to be global?

Reply:
I need the API result object in PostRunE and hence made it global. Open to suggestions :)

Reply:
to share state between methods of a command we usually scope them to the struct (`CreateOpts`) but for now this should be ok